dbt_meshify/main.py

```diff
@@ -51,3 +51,26 @@ def split():
         holder.register_project(subproject)
 
     print(holder.project_map())
+
+
+@cli.command(name="contract")
+@click.option(
+    "--select",
+    "-s"
+    )
+@click.option(
+    "--exclude",
+    "-e"
+    )
+@click.option(
+    "--project-path",
+    default="."
+)
+def contract(select, exclude, project_path):
+    path = Path(project_path).expanduser().resolve()
+    project = DbtProject.from_directory(path)
+    resources = list(project.select_resources(select=select, exclude=exclude, output_key="unique_id"))
+    models = [json.loads(resource)["unique_id"] for resource in resources if json.loads(resource)["unique_id"].startswith("model")]
```

Comment:
This seems a little cumbersome. Perhaps parsing json results into `List[Dict]` should be a responsibility of `select_resources`?

Also, once you have an iterable (like List or Set) you can use the `filter()` function with a `lambda` to filter the list down to values starting with `"model"`.

Something like this, assuming resources is `List[Dict[str, str]]`.
```python
models = filter(lambda x: x['unique_id'].startswith('model'), resources)
```

Reply:
agreed here as well! i was thinking the `output_key` argument would only ever be one thing (for now at least), so the filtering and reducing can be housed a layer down for sure -- i don't love the `json.loads()` here either, but i think the output from the ls is a little wonky when it comes back as json!